### PR TITLE
Modified README.md to point to Kubeflow.org community page. Issue #1237

### LINKED ISSUE #1237: "Getting Involved" in README.md should point to kubeflow.org
https://github.com/kubeflow/kubeflow#get-involved should now point to https://www.kubeflow.org/docs/about/community/

Related issue https://github.com/kubeflow/website/issues/93.

Since the README.md is now becoming quite small, the Table of Contents should probably be removed.

### PR DESCRIPTION
**Pull request by amsaha**
Please take a look at #1237 to see how the README would look with this patch.

Fix #1237

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1247)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,15 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+<img src="https://www.kubeflow.org/images/logo.svg" width="100">
```
Comment (gaocegege) [inline README.md]: I think we could make the image in the center like 

```html
<h1 align="center">
	<br>
	<img width="400" src="docs/images/logo.jpeg" alt="cyclone">
	<br>
</h1>
```

Comment (amsaha) [reply README.md]: I tried that but it does not look good :-(

screen shot 2018-07-20 at 8 31 13 pm

**Comment by amsaha** [comment]
@gaocegege Tried centering but it not look good:

screen shot 2018-07-20 at 8 31 13 pm

**Comment by amsaha** [comment]
/hold 

Till we get clarification on centering or not, lets not merge.

**Comment by gaocegege** [comment]
/cc @jlewi 

@amsaha Personally I prefer the latter 🤔

**Comment by amsaha** [comment]
As a data point, https://github.com/kubernetes/kubernetes/blob/master/README.md does it left justified. With the center justification the logo starts moving around with resizing of the browser window. 

Anyway, others comment please.

**Comment by jlewi** [comment]
Thank you.

This is a significant improvement in that it directs people to our website so we don't have to deal with information in two places.

Left justifying the logo looks fine to me. If people have a strong opinion we can fix it in a follow up issue. I don't want to block the other changes on that. So canceling the hold and approving.

/hold cancel
/lgtm
/approve

**Comment by amsaha** [comment]
yes, let me do that.

**Comment by amsaha** [comment]
@gaocegege @jlewi rebasing worked. Needs lgtm label again.

**Pull request merged**